src/App.tsx

```diff
@@ -212,13 +214,9 @@ function App() {
                 <ValueEditor
                   key={`val${index}`}
                   value={value}
-                  index={index + 1}
+                  index={index}
```

Comment:
I'm passing the unmodified index because I need to use it when clearing a specific ValueEditor

Reply:
That code was so bad! Thank you for correcting me